index.js

```diff
@@ -11,6 +11,7 @@ const colors = {
   blue: '#7FDBFF',
   lightGray: '#DDDDDD',
   gray: '#AAAAAA',
+  orange: '#FF851B',
```

Comment:
Just tested it and I feel orange and red might be too visually similar, could you perhaps change it to be yellow? `#FFDC00` - thanks! :tada: